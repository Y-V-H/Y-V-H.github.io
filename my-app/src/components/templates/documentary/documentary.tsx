import React from "react";
import { Header } from '../../atoms/header/header'

export const Documentary = () => {
    return (
        <Header size='h1' children="Documentary page"/>
    )
}