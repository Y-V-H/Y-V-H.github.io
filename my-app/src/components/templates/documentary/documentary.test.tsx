import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom'
import { Documentary } from './documentary';

test('should render Comedy template', () => {
    const component = renderer.create(
        <Documentary />
    );
    expect(component.toJSON()).toMatchSnapshot();
});