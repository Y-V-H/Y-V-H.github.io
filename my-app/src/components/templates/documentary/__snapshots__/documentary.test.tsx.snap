// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Comedy template 1`] = `
<h1
  className="a-header a-header--h1 undefined"
>
  Documentary page
</h1>
`;
